common/build-style/cargo.sh

```diff
@@ -0,0 +1,33 @@
+#
+# This helper is for building rust projects which use cargo for building
+#
+
+do_configure() {
+	mkdir -p .cargo
+	# Don't be verbose, respect makejobs
+	cat >> .cargo/config <<EOF
+[term]
+verbose = false
+[build]
+jobs = ${makejobs#*j}
+EOF
+}
+
+do_build() {
+	: ${make_cmd:=cargo}
+
+	${make_cmd} build --release ${configure_args}
```

Comment:
shouldn't it be make_build_args ?

Reply:
Hm, we can't do configuring in do_configure with cargo (AFAIK), so we sort of do configuring in do_build (e.g. enabling/disabling features with --features), so configure_args seems like the right way to go, even though it's not used in do_configure.